util/src/main/java/tc/oc/pgm/util/inventory/InventoryUtils.java

```diff
@@ -188,6 +189,40 @@ default void setUnbreakable(ItemStack item, boolean unbreakable) {
     void applyAttributeModifiers(
         SetMultimap<Attribute, AttributeModifier> modifiers, ItemMeta meta);
 
+    boolean attributesEqual(ItemMeta meta1, ItemMeta meta2);
+
+    default boolean modifiersDiffer(
+        Collection<AttributeModifier> a, Collection<AttributeModifier> b) {
```

Comment:
nit: style: imo this method is going too far with shortening variables names and makes it a bit harder to follow

Admittedly its less of a problem here given that what the method does is pretty simple

Reply:
the method is rather trivial, so it's either A and B or modifiersA modifiersB, or modifiers1, modifiers2

given how trivial it is i don't think it matters much